# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1300514298 25200
#      Fri Mar 18 22:58:18 2011 -0700
# Node ID 33f7d8f95f5bb28e79fce20c711f16c08d41204c
# Parent  33bbae85769aac5da079b77fb51400f3f5cd20ca
Add nth_element function to documentation.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2010-03-18  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/matrix.txi: Add nth_element function to documentation.
+
 2010-03-18  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/doccheck/aspell-octave.en.pws, interpreter/io.txi,
 	interpreter/octave.texi, interpreter/plot.txi: Add uimenu, uigetdir,
 	uigetfile, and uiputfile to documentation.
 
 2010-03-18  Rik  <octave@nomad.inbox5.com>
 
diff --git a/doc/interpreter/matrix.txi b/doc/interpreter/matrix.txi
--- a/doc/interpreter/matrix.txi
+++ b/doc/interpreter/matrix.txi
@@ -136,16 +136,18 @@ a = [1, 2; 2, 3; 3, 1];
 [s, i] = sort (a (:, 2));
 a (i, :)
      @result{}  3  1
          1  2
          2  3
 @end group
 @end example
 
+@DOCSTRING(nth_element)
+
 @anchor{doc-triu}
 @DOCSTRING(tril)
 
 @DOCSTRING(vec)
 
 @DOCSTRING(vech)
 
 @DOCSTRING(prepad)
